Explorer/Assets/AddressableAssetsData/AssetGroups/UI.asset

```diff
@@ -27,11 +27,6 @@ MonoBehaviour:
     m_ReadOnly: 0
     m_SerializedLabels: []
     FlaggedDuringContentUpdateRestriction: 0
-  - m_GUID: dbfb7e67a438449f9a887f820f733868
-    m_Address: PopupCloser
```

Comment:
What happened to the popup closer?

Reply:
hmmmm probably a mistake, I had a conflict here